torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -40,7 +40,7 @@ std::vector<IValue> runNode(Node* n) {
         if (t.requires_grad()) {
           throw c10::Error("Can't insert requires grad as constant", "");
         }
-        return IValue(autograd::as_variable_ref(t).data());
+        return IValue(t);
```

Comment:
It's okay to remove the `.data()` call here, because the check above this line throws error if `t` requires grad, so we know that `t` is guaranteed to not require grad at this point.